src/verifier_test.cc

```diff
@@ -142,6 +145,58 @@ TEST_F(VerifierTest, ProbeFrameBuffer) {
   EXPECT_TRUE(r.IsSuccess());
 }
 
+TEST_F(VerifierTest, ProbeFrameBufferNotRect) {
+  uint8_t frame_buffer[10][10][4] = {};
+
+  frame_buffer[2][1][0] = 128;
+  frame_buffer[2][1][1] = 64;
+  frame_buffer[2][1][2] = 51;
+  frame_buffer[2][1][3] = 204;
+
+  frame_buffer[3][7][0] = 51;
+  frame_buffer[3][7][1] = 204;
+  frame_buffer[3][7][2] = 64;
+  frame_buffer[3][7][3] = 128;
+
+  ProbeCommand probe;
+  probe.SetIsRGBA();
+  probe.SetX(1.0f);
+  probe.SetY(2.0f);
+  probe.SetR(0.5f);
+  probe.SetG(0.25f);
+  probe.SetB(0.2f);
+  probe.SetA(0.8f);
+
+  Verifier verifier;
+  Result r = verifier.Probe(&probe, 4, 40, 10, 10,
+                            static_cast<const void*>(frame_buffer));
+  EXPECT_TRUE(r.IsSuccess());
+
+  probe.SetIsRGBA();
+  probe.SetX(7.0f);
+  probe.SetY(3.0f);
+  probe.SetR(0.2f);
+  probe.SetG(0.8f);
+  probe.SetB(0.25f);
+  probe.SetA(0.5f);
+
+  r = verifier.Probe(&probe, 4, 40, 10, 10,
+                     static_cast<const void*>(frame_buffer));
+  EXPECT_TRUE(r.IsSuccess());
```

Comment:
Does this second Probe call verify anything different from the one above?

Reply:
Removed